src/hotspot/share/adlc/formssel.cpp

```diff
@@ -778,6 +778,7 @@ bool InstructForm::captures_bottom_type(FormDict &globals) const {
        !strcmp(_matrule->_rChild->_opType,"RotateLeft")   ||
        !strcmp(_matrule->_rChild->_opType,"RotateRight")   ||
 #if INCLUDE_SHENANDOAHGC
+       !strcmp(_matrule->_rChild->_opType,"StrInflatedCopy") ||
```

Comment:
Should not be inside the ifdef.

Reply:
Yes, good catch, thank you! (That happens when doing last-minute re-ordering ...). Fixed.